var/spack/repos/builtin/packages/py-jax/package.py

```diff
@@ -7,7 +7,7 @@
 from spack.package import *
 
 
-class PyJax(PythonPackage, CudaPackage):
+class PyJax(PythonPackage):
```

Comment:
JAX is pure-Python and doesn't use CUDA. If you want to control which cuda_arch is used by jaxlib, you should use:
```console
$ spack install py-jax ^py-jaxlib +cuda cuda_arch=...
```

Reply:
I would think that this kind of default install would surprise most users, no? also, if we keep this, alphafold should probably be addressed.

Reply:
What kind of default install? We can tell py-jaxlib to build +cuda by default if you want.

Reply:
Is it the case that alphafold requires `py-jax ^py-jaxlib+cuda`? We can add a fake dep on `py-jaxlib` if that helps. I try to avoid variant forwarding, especially now that we have `++` and `==` variants in the spec syntax.

Reply:
I think jaxlib +cuda by default is reasonable...

Reply:
Should alphafold also default to +cuda?